pkgs/tools/misc/grub/pvgrub_image/configs/grub.cfg

```diff
@@ -0,0 +1,9 @@
+if search -s -f /boot/grub/grub.cfg ; then
+	echo "Reading (${root})/boot/grub/grub.cfg"
```

Comment:
The parentheses and ${root} might make sense in a larger context, but they don't to me in this small context.